zenodo/modules/github/utils.py

```diff
@@ -126,25 +126,23 @@ def sync(gh, extra_data, sync_hooks=True):
         depositions=[],
         errors=None,
     )
-    login = extra_data["login"]
 
     existing_set = set(extra_data['repos'].keys())
     new_set = set()
 
-    for u in [login] + [x.login for x in gh.iter_orgs()]:
-        for r in gh.iter_user_repos(u, type='owner', sort='full_name'):
-            # Add if not in list
-            if r.full_name not in extra_data["repos"]:
-                extra_data["repos"][r.full_name] = copy.copy(repo_data)
-            # Update description
-            extra_data["repos"][r.full_name]["description"] = r.description
-            new_set.add(r.full_name)
-
-            # # TODO:
-            # if sync_hooks:
-            #     for h in r.iter_hooks():
-            #         if h.name == 'web' and 'url' in h.config:
-            #             h.config['url'] ==
+    for r in gh.iter_repos(type='all', sort='full_name'):
```

Comment:
Can I get you to rebase and fix this line into:

```python
    repos = gh.iter_repos(type='all', sort='full_name')
    repos.headers['Accept'] = 'application/vnd.github.moondragon+json'

    for r in repos:
        # Next line to be replaced with "r.permissions['admin']" once
        # fix has been integrated in github3.py
        if r.to_json()['permissions']['admin']:
```

I've verified that it works, so then I'll be able to merge.